packages/arm-batch-rest/package.json

```diff
@@ -0,0 +1,110 @@
+  "main": "./lib-cjs/index.js",
+  "module": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Azure/BatchExplorer"
+  },
+  "files": [
+    "lib",
+    "lib-cjs"
+  ],
+  "engines": {
+    "node": ">=14.0.0"
+  },
+  "test": "jest",
+  "scripts": {
+    "build": "npm run compile",
+    "build:clean": "run-s clean compile",
+    "build:test": "run-s build test",
+    "compile": "run-p compile:*",
+    "compile:esm": "tsc -b ./config/tsconfig.build.json",
+    "compile:cjs": "tsc -b ./config/tsconfig.cjs.json",
+    "clean": "run-p clean:*",
+    "clean:build": "bux rmrf ./build",
+    "clean:esm": "bux rmrf ./lib",
+    "clean:cjs": "bux rmrf ./lib-cjs",
+    "generate:client": "autorest --typescript swagger/README.md && npm run format",
+    "lint": "eslint",
+    "test": "jest",
+    "test:all": "npm run test:coverage",
+    "test:coverage": "jest --collect-coverage",
+    "test:watch": "jest --watch",
+    "watch": "npm run compile:esm -- --watch --preserveWatchOutput",
+    "workspace:build": "npm run build",
+    "workspace:build:clean": "npm run build:clean",
+    "workspace:build:desktop": "npm run build",
+    "workspace:build:lib": "npm run build",
+    "workspace:build:web": "npm run build",
+    "workspace:build:package": "npm run build:clean",
+    "workspace:build:prod": "npm run build:clean",
+    "workspace:build:test": "npm run build:test",
+    "workspace:clean": "npm run clean",
+    "workspace:launch:desktop": "npm run watch",
+    "workspace:launch:web": "npm run watch",
+    "workspace:lint": "npm run lint",
+    "workspace:lint:fix": "npm run lint -- --fix",
+    "workspace:start:desktop": "npm run watch",
+    "workspace:start:web": "npm run watch",
+    "workspace:test": "npm run test",
+    "workspace:test:all": "npm run test:all",
+    "workspace:test:desktop": "npm run test",
+    "workspace:test:lib": "npm run test",
+    "workspace:test:lib:all": "npm run test:all",
+    "workspace:test:web": "npm run test",
+    "workspace:test:web:all": "npm run test:all"
+  },
+  "sideEffects": false,
+  "autoPublish": false,
+  "dependencies": {
+    "@azure-rest/core-client": "1.0.0-beta.10",
+    "@azure/core-auth": "^1.3.0",
+    "@azure/core-lro": "^2.2.0",
+    "@azure/core-paging": "^1.2.0",
+    "@azure/core-rest-pipeline": "^1.8.0",
+    "tslib": "^2.2.0",
+    "@batch/ui-common": "^1.0.0"
+  },
+  "devDependencies": {
+    "@batch/common-config": "^1.0.0",
+    "@azure/identity": "^2.0.1",
+    "@types/chai": "^4.2.8",
+    "@types/jest": "^29.2.4",
+    "@types/node": "^14.0.0",
+    "@types/node-fetch": "^2.6.2",
+    "autorest": "latest",
+    "chai": "^4.2.0",
+    "jest": "^29.3.1",
+    "jest-junit": "^12.2.0",
+    "mkdirp": "^1.0.4",
+    "node-fetch": "^2.6.7",
+    "nyc": "^14.0.0",
+    "source-map-support": "^0.5.9",
+    "ts-jest": "^29.0.3",
+    "ts-node": "^10.9.1",
+    "typescript": "~4.9.4"
+  },
+  "//metadata": {
```

Comment:
This is for tracking version information within the SDK. Since we are not technically going to be correlating the package version of the module and the RLC package # defined in the swagger/README.md, we can remove this.